presentation/components/src/presentation-components/instance-filter-builder/PresentationInstanceFilterBuilder.tsx

```diff
@@ -86,25 +88,86 @@ export function usePresentationInstanceFilteringProps(descriptor: Descriptor, cl
     });
   }, [classes, propertyInfos, classHierarchyProvider, enableClassFiltering]);
 
+  const propertyRenderer = React.useCallback((name: string) => {
+    const instanceFilterPropertyInfo = propertyInfos.find((info) => info.propertyDescription.name === name);
+    assert(instanceFilterPropertyInfo !== undefined);
+    return <PresentationInstanceFilterProperty instanceFilterPropertyInfo={instanceFilterPropertyInfo} />;
+  }, [propertyInfos]);
+
   return {
     onPropertySelected,
     onClearClasses,
     onClassDeselected,
     onClassSelected,
+    propertyRenderer,
     properties,
     classes,
     selectedClasses,
   };
 }
 
+interface CategoryTooltipContentProps {
+  instanceFilterPropertyInfo: InstanceFilterPropertyInfo;
+}
+
+function CategoryTooltipContent(props: CategoryTooltipContentProps) {
+  const { instanceFilterPropertyInfo } = props;
+  const [schemaName, className] = instanceFilterPropertyInfo.className.split(":");
+  return <table>
+    <tbody>
+      <tr>
+        <th className="tooltip-content-header">CATEGORY: </th>
```

Comment:
The labels should probably be localized.

Reply:
Fixed it.